Shared/css/style.css

```diff
@@ -5088,9 +5088,20 @@ textarea#filecont{
 }
 
 .resultedbuttons{
-  margin: 7px;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  margin: 20px 2px 4px 2px;
   padding-left: 10px;
   padding-right: 10px;
   background-color: #614875;
   color: #ffffff;
+  width: 110px;
+  height: 30px;
+  border: 0px;
+  font-size: 14px;
+  transition: 0.1s background-color;
+  border-radius: 4px;
+}
+
+.resultedbuttons:hover{
+  filter: brightness(1.05); 
```

Comment:
```suggestion
  filter: brightness(1.2); 
```

Reply:
I agree. Tested it and implemented the change! https://https://c17alepe.webug.his.se:20002/DuggaSys/resulted.php?cid=2&coursevers=97732
if you wanna see